tests/framework/installer/ceph_manifests.go

```diff
@@ -983,7 +1023,7 @@ spec:
         valueFrom:
           secretKeyRef:
             name: rook-ceph-mon
-            key: admin-secret
+            key: admin
```

Comment:
@rootfs I'd rather leave the toolbox to use the `admin-secret` key so we don't have an issue during upgrade where the toolbox doesn't work. When the secret is written, can we write the key to both `admin` and `admin-secret` to ensure upgrade compatibility?

Reply:
sounds good, let me find where the secret is written